en/docs/guides/access-delegation/configure-jwt-grant.md

```diff
@@ -49,7 +49,7 @@ This page guides you through the flow involved in exchanging a JWT assertion wit
 
 ## Configure IAT 
 
-Optionally you can configure the IAT (issued at) claim. It identifies the time at which the JWT was issued and can be used to determine the age of the JWT.
+**Optionally** you can configure the IAT (issued at) claim. It identifies the time at which the JWT was issued and can be used to determine the age of the JWT.
```

Comment:
Since this is the first line of the page, shall we change it into something along the lines of, 

While configuring the JWT grant, you can optionally configure the IAT (issued at) claim.

Reply:
This is not the first line of the page...this section comes somewhere in the middle. line 52